Bundle/WidgetBundle/Resolver/WidgetResolver.php

```diff
@@ -77,7 +80,10 @@ protected function assert($value, $operator, $expected)
         $result = false;
         switch ($operator) {
             case self::OPERAND_EQUAL:
-                $result = $value === $expected;
+                $result = $value == $expected;
```

Comment:
The `$value` variable isn't always a boolean, we cannot cast is in bool.
I think our problem here is deeper and we need to dig more for a good architecture.
What do we do with this PR ? There isn't any BC break but there is a workaround by using the `OPERAND_TRUE` instead the `OPERAND_EQUAL` for now so... we can take our time if you prefer @paulandrieux